README.rst

```diff
@@ -1,32 +1,36 @@
+**WARNING**: This repository will eventually be obsolete. The content will
+exist under the `dwave-examples <https://github.com/dwave-examples>`_ account.
+
 Demos
 =====
 A collection of demos using the D-Wave Ocean SDK.
 
-**NOTE** Each demo is stored in its own subdirectory and may have its own
-set of package requirements (for example, `antenna-selection/requirements.txt
-<antenna-selection/requirements.txt>`_).
+**NOTE** Each demo is stored in its own submodule and may have its own
+set of package requirements (e.g. `antenna-selection/requirements.txt
+<https://github.com/dwave-examples/antenna-selection/blob/master/requirements.txt>`_).
 
 Setup Instructions
 ------------------
-As each demo may have a different set of package requirements, you may want to
-set up a `virtual environment <https://docs.ocean.dwavesys.com/en/latest/overview/install.html#python-virtual-environment>`_
+As each demo may have a different set of package requirements, you may
+want to set up a `virtual environment <https://docs.ocean.dwavesys.com/en/latest/overview/install.html#python-virtual-environment>`_
 to contain said packages and run your demo within that environment.
 
 Package Installation
 ~~~~~~~~~~~~~~~~~~~~
 ::
 
-  git clone https://github.com/dwavesystems/demos.git
-  cd demos/<desired-demo>
-  pip install -r requirements.txt
+  git clone --recursive https://github.com/dwavesystems/demos.git
+  cd demos/
+  git submodule foreach git pull origin master
+  pip install -r <desired-demo>/requirements.txt
 
 Please note that to run demos that access a D-Wave system, you must:
 
 * Sign up for `D-Wave Leap <https://cloud.dwavesys.com/leap/signup/>`_ in order
   to get an authentication token
 * Create a `configuration file <https://docs.ocean.dwavesys.com/en/latest/overview/dwavesys.html#configuring-a-d-wave-system-as-a-solver>`_,
-  so that you can easily submit your problems to a D-Wave system without your
-  authentication token being stored within your code
+  so that you can easily submit your problems to a D-Wave system without
+  storing your authentication token in your code.
```

Comment:
You don't need a config file for that. Just one environment variable! :smiley:

Reply:
Yes, I believe I understand what you're saying. Just store my token in an environment variable in something like `DWAVE_TOKEN`, correct?

Would the user also have to store their API endpoint in their environment variable as well? I'm asking because I want to know what the `dwave config create` helps the user with and what would happen if the user decides to deal with all of this themselves. Thanks!

Reply:
That's well documented in [intro](https://docs.ocean.dwavesys.com/projects/cloud-client/en/latest/intro.html#configuration) and [config](https://docs.ocean.dwavesys.com/projects/cloud-client/en/latest/reference/configuration.html) sections of the cloud client docs.

TL;DR variant would be: for most users (all Leap users) defining `DWAVE_API_TOKEN` is enough.

Reply:
Added environment variables as an alternative option